jina/parsers/peapods/pod.py

```diff
@@ -63,12 +63,14 @@ def mixin_base_pod_parser(parser):
         else argparse.SUPPRESS,
     )
 
-    # hosts for peas
+    # hosts for peas, when parallel greater than 1 each pea should have it's own host
 
     gp.add_argument(
         '--peas-hosts',
         type=KVAppendAction,
         metavar='KEY=VALUE',
         nargs='*',
-        help='The hosts of the pea when parallel greater than 1'
+        help='''The hosts of the peas when parallel greater than 1,
```

Comment:
Each Pea has a pod is not true, I think the Peas not present in this map will get the `host` from the `Pods` arguments, as well as Head and Tail Peas

Reply:
Key is not the host id, key is the `pea_id`